### PR TITLE
Added timestamps

### PR DESCRIPTION
**Pull request by Chronojam**

### CODE REVIEW

**Comment by swagatha-christie** [comment]
By analyzing the blame information on this pull request, we identified @oliverwoodings and @roberttod to be potential reviewers

**Comment by oliverwoodings** [comment]
Great idea, thanks! Couple of issues we need to resolve, however:

- driftwood is designed to go in smartserve (the bundle of code that goes on all our client sites). moment is pretty huge, so we can't pull it in. We should just use the native JS date object.
- It might be more useful to emit the time at a composite log level, rather than the console level. This means that all registered loggers will receive the same timestamp for the log, instead of having to create their own. You could do that [here](https://github.com/QubitProducts/driftwood/blob/master/src/createLogger.js#L11) (I suggest we create a JS date object there and pass it to each logger. The individual loggers can then decide how they want to render it).

**Comment by Chronojam** [comment]
Ive added some support for formatting strings, 
I imagine you could initialize your loggers like this
```
var log = createLogger('mymodule-main', 'DATE - NAME: [LEVEL] MESSAGE')
```

File: src/createLogger.js [hunk 1]
```diff
@@ -8,9 +8,10 @@ function noop () { }
 
 function compositeLogger (name, loggers) {
   return function (name, level, message, metadata) {
+    var date_now = new Date()
```
Comment (oliverwoodings) [inline src/createLogger.js]: We use camelCase everywhere

**Comment by oliverwoodings** [comment]
Hmm, interesting one. So i'm a bit fan of making things as customisable as possible, but we really do have to be stingy when it comes to adding additional features to stuff that goes into smartserve. Could we not just find a formatting structure that we like and stick with it? How often would someone really want to customise this?

**Pull request closed without merging**